### ISSUE TITLE
DHCP Range is Cut Off in New Chronometer

### ISSUE THREAD

**Issue by jpmck** [pi-hole/pi-hole#1542]
**In raising this issue, I confirm the following (please check boxes, eg [X]) Failure to fill the template will close your issue:**

- [x] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [X] The issue I am reporting can be *replicated*
- [X] The issue I am reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)).

**How familiar are you with the codebase?:**
10

---
**[BUG REPORT]:**
The DHCP range is cut off by the second column of information provided in the updated Chronometer.

Issue lies with line [368 of chronometer.sh](https://github.com/pi-hole/pi-hole/blob/ecde2225122904a89543712ec8f2b0fd1da26ff6/advanced/Scripts/chronometer.sh#L368)

**[BUG | ISSUE] Expected Behaviour:**
Expected to see the entire range:
```
     DHCP: 192.168.100.100 to 192.168.100.150
```

**[BUG | ISSUE] Actual Behaviour:**
DHCP range is cut off by second column:
```
     DHCP: 192.168.100.100 to 1(Leased: 12 of 51)
```
![image](https://user-images.githubusercontent.com/6764712/27363711-19e83422-5604-11e7-9e32-be3cccb4947e.png)

**[BUG | ISSUE] Steps to reproduce:**
- Run ```pihole -c```

_This template was created based on the work of [`udemy-dl`](https://github.com/nishad/udemy-dl/blob/master/LICENSE)._

**Comment by WaLLy3K** [comment]
I suspect that this should be fixed when #1518 is released, but I'll definitely look into the issue further.

**Comment by jacobsalmela** [comment]
This is fixed in v3.2 (#1518)
